service/src/main/java/at/jku/isse/ecco/storage/mem/repository/MemRepository.java

```diff
@@ -12,37 +12,45 @@
 import at.jku.isse.ecco.storage.mem.feature.MemFeature;
 import at.jku.isse.ecco.storage.mem.module.MemModule;
 import org.eclipse.collections.impl.factory.Maps;
+//import org.springframework.stereotype.Component;
 
 import java.util.*;
 
 /**
  * Memory implementation of {@link Repository}.
  */
+//@Component
```

Comment:
remove dead code

Reply:
again, springboot should not be necessary for dependencies. Imagine if you hadn't access to the source of the dependencies

Reply:
Done